pkg/rpcclient/wsclient_test.go

```diff
@@ -276,6 +276,7 @@ func TestWSClientEvents(t *testing.T) {
 	require.False(t, ok)
 	_, ok = <-ntfCh
 	require.False(t, ok)
+	t.Cleanup(srv.Close)
```

Comment:
> to prevent fd leak.

Have you checked that it's fd leak? Because I'm not sure about it. If you're not sure, then either don't write it or check for sure.

Reply:
It's just written in test HTTP server constructor that it should be properly closed in the end of the test.